src/components/Cohort/Documents/DocumentList/DocumentList.tsx

```diff
@@ -183,10 +189,26 @@ const DocumentRow: React.FC<DocumentRowTypes> = ({
             </Grid>
           </Grid>
           <Grid container item xs={1} justify="center">
-            <IconButton onClick={() => openPdfDialog(row.id)}>
-              <PdfIcon height="30px" fill="#ED6D91" />
-            </IconButton>
-
+            {row.content && row.content[0] ? (
+              row.content[0].attachment?.url?.endsWith('.pdf') ? (
+                <IconButton onClick={() => openPdfDialog(row.id)}>
+                  <PdfIcon height="30px" fill="#ED6D91" />
+                </IconButton>
+              ) : (
+                <>
+                  <IconButton type="button" onClick={handleImageOpen}>
+                    <ImageIcon />
+                  </IconButton>
+                  <Modal open={isImageOpen} onClose={handleImageClose}>
+                    <img
+                      className={classes.img}
+                      src={`${FILES_SERVER_URL}${row.content[0].attachment?.url?.replace(/^file:\/\//, '')}`}
```

Comment:
sometimes you use f-strings sometimes you concatenate with `+`, can you do the same everywhere ? (I personnaly like f-strings better but you choose)